metrics_sdk/lib/opentelemetry/sdk/metrics/instrument/counter.rb

```diff
@@ -38,12 +31,6 @@ def add(increment, attributes: {})
             OpenTelemetry.handle_error(exception: e)
             nil
           end
-
-          private
-
-          def default_aggregation
-            OpenTelemetry::SDK::Metrics::Aggregation::Sum.new
-          end
```

Comment:
I've moved the default aggregations definition to Meter because it uses them to create MetricStream there. Thinking again, leaving definitions in each instrument class may look better but it would have to be a public method.